packages/client/lib/CredentialOffer/CredentialOfferClient.ts

```diff
@@ -0,0 +1,60 @@
+import { CredentialOfferWithBaseURL, EndpointMetadata, OpenId4VCIVersion } from '@sphereon/openid4vci-common';
+import Debug from 'debug';
+
+import { CredentialIssuanceClient } from './CredentialIssuanceClient';
+
+const debug = Debug('sphereon:openid4vci:credentialOffer');
+
+export class CredentialOfferClient implements CredentialIssuanceClient {
+  public readonly _version: OpenId4VCIVersion;
+  private readonly _credentialOfferWithBaseURL: CredentialOfferWithBaseURL;
+
+  public constructor(issuanceInitiationURI: string, credentialOfferWithBaseURL: CredentialOfferWithBaseURL) {
+    this._version = OpenId4VCIVersion.VER_11;
+    this._credentialOfferWithBaseURL = credentialOfferWithBaseURL;
+  }
+
+  public static fromURI(credentialOfferURI: string): CredentialOfferClient {
+    // FIXME implement the function
+    debug(`'fromURI' is not implemented yet: ${credentialOfferURI}`);
+    throw new Error(`'fromURI' is not implemented yet: ${credentialOfferURI}`);
+  }
+
+  public static async getServerMetaData(credentialOfferWithBaseURL: CredentialOfferWithBaseURL): Promise<EndpointMetadata> {
+    throw new Error(`getServerMetaData': Not implemented yet.`);
+  }
+
+  public static getCredentialTypes(): string[] {
+    return [];
+  }
+
+  public static getIssuer(): string {
+    return '';
+  }
+
+  public static assertIssuerData(credentialOfferWithBaseURL: CredentialOfferWithBaseURL): void {
```

Comment:
Why would this be a static on a class with state that is not responsible for it to begin with?